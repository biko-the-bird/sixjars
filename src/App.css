.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 20vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(80px + 2vmin);
  color: white;
  border-bottom: 10px dashed white;
}


.footer-line {
  text-align: center;
  margin-right: 3%;
  margin-left: 3%;
}

.App-link {
  color: #61dafb;
}

.half-line {
  width: 75%;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
