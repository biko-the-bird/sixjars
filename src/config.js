
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

//this is a configuration for enzyme that needs
//to be imported in each testing file
const config = configure({ adapter: new Adapter() });
export default config;