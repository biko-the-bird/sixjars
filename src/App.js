import React from 'react';
import logo from './logo.svg';
import './App.css';
import Balance from './components/balance';

function App() {
  return (
    <div className="App">
      <header className="App-header">
      Six Jars
      </header>
      <Balance/>
    </div>
  );
}

export default App;
