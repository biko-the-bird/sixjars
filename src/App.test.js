import React from 'react';
import config from './config';
import {shallow} from 'enzyme';
import App from './App';

describe('App', () => {
  const app = shallow(<App/>);
  it("renders on the page", () => {
    expect(app).toMatchSnapshot();
  })
});
