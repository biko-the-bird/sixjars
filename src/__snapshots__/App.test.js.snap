// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders on the page 1`] = `ShallowWrapper {}`;
